### DISCUSSION TITLE
How to use position: fixed or position: sticky?

### DISCUSSION THREAD

**Discussion by davidstellini** [idiotWu/smooth-scrollbar#473] [Q&A]
Since this is using translate3d, position fixed or sticky won't work inside it.
Is there any way to force it to use native scrolling?

**Comment by davidstellini** [comment]
I've seen this, but it's not a valid solution in my case: https://github.com/idiotWu/smooth-scrollbar/discussions/362#discussioncomment-854090

I have a bunch of animations that alternate between fixed and relative positions, moving them in and out of the app root, where the scrollbar is, is really nasty imo, I would much rather figure out a way to do native scrolling.

**Reply by dolphin-wood** [reply]
Just curious what kind of animation needs to alternate between fixed and relative positions. Is there any demo?

**Comment by bradleyhodges** [comment]
I have prepared a solution to this issue for Node.js users: https://github.com/idiotWu/smooth-scrollbar/discussions/362#discussioncomment-10345508
